docs/docs/getting-started/02-oauth-tutorial.mdx

```diff
@@ -176,51 +202,73 @@ The callback URL we insert should have the following pattern:
 [origin]/api/auth/callback/[provider]
 ```
 
-In this case, given we want to try our authentication working locally on our machine and we're using **Github** as our OAuth provider, it'll be:
+In this case, given we want to try our authentication working locally on our machine and we're using **GitHub** as our OAuth provider, it'll be:
+
+<Tabs groupId="frameworks">
+  <TabItem value="next" label="Next.js" default>
 
 ```
 http://localhost:3000/api/auth/callback/github
 ```
 
```

Comment:
Let's add a note here where the last portion of the URL (`github`) comes from, and what is the relation with `signIn("github")`.

Not sure what's the best way to document this, but it's the provider configuration's `id` property.